### ISSUE TITLE
Edit dashboard, change structure data binding

### ISSUE THREAD

**Issue by umeshv** [angular-dashboard-framework/angular-dashboard-framework#38]
Hi @sdorra,

I am trying to enhance edit dashboard change structure radio button with pictorial representation of layout using div. However, when I select this individual div element the layout structure changes but the selected div is not highlighted. 

I found that this not happening because $scope.changeStructure function is not setting the model.structure with the selected structure name. This can be done by setting model.structure = name after line 177 in dashboard.js.

My question is can we include this change in main branch as this will help us to reflect the correct selected structure after structure change?

Thanks a lot for the help.

**Comment by harshmer** [comment]
Hi @umeshv 

Firstly, you did a good work in changing the representation.

I would like to know how did you implement the pictorial representation?
Could you please help me out with that?
I want to implement the same thing. If you can post your solution to me at harshmer@yahoo.com? please?

Thanks,
Awaiting your response.

Regards,
Harsh

**Comment by umeshv** [comment]
Hi @harshmer 

You can change the structure name and can create the template with the same name. Also edit the dashboard-edit template to use anchor tag instead of radio button inside ng-repeat which will give you a desired result.

Thanks.

**Comment by Macking** [comment]
How can I define the nest structure, such as 3-9 (12(6-6)/6-6)/6-6?

**Comment by harshmer** [comment]
Hi @Macking :
I didnt understand your question. Can you describe in brief please?

**Comment by Macking** [comment]
At first I want to define complex structure for my project. So I believe the nest structure can be customized as 3-9 (12(6-6)/6-6)/6-6 against the pre-define structure. But it's so hard to parse this string. At last, I find a bootstrap online site to generate structure, then I get the structure string such as:
```
<div class="container">
  <div class="row">
    <div class="col-md-6">
      <div class="row">
        <div class="col-md-8">
        </div>
        <div class="col-md-4">
        </div>
      </div>
    </div>
    <div class="col-md-6">
    </div>
  </div>
  <div class="row">
      <div class="col-md-12">
      <div class="row">
         <div class="col-md-6">
         </div>
        <div class="col-md-6">
        </div>
      </div>
    </div>
  </div>
</div>
```
and I write some code to translate into:
```
{"rows":[
  {"columns":[
      {"styleClass":"col-md-6","rows":[
         {"columns":[
             {"styleClass":"col-md-8"},{"styleClass":"col-md-4"}]
           }]
        },
        {"styleClass":"col-md-6"}]},
  {"columns":[
     {"styleClass":"col-md-12","rows":[
        {"columns":[{"styleClass":"col-md-6"},{"styleClass":"col-md-6"}]
      }]
    }]
  }]
}
```
 and it can be used in ADF as a structure.
So please ignore my last question. Thank you @harshmer

**Comment by Noman-Malik** [comment]
is we adjust column width dynamically according to widget
